### PR TITLE
Dependency upgrades

### PR DESCRIPTION
**Pull request by bergmark**

### CODE REVIEW

File: tests/common/mock.rs [hunk 1]
```diff
@@ -222,4 +222,56 @@ impl CloudWatch for MockCloudWatchClient {
...
+    }
+
+    async fn list_metric_streams(
+        &self,
+        input: ListMetricStreamsInput,
+    ) -> Result<ListMetricStreamsOutput, RusotoError<ListMetricStreamsError>> {
+        todo!()
+    }
+
+    fn put_metric_stream<'life0, 'async_trait>(
```
Comment (bergmark) [inline tests/common/mock.rs:247]: I couldn't figure out how to make a prettier type sig out of this one...

**Comment by bergmark** [comment]
ping @Marwes

**Review by Marwes — APPROVED** [review]

**Comment by bergmark** [comment]
can you please make a release?

**Comment by Marwes** [comment]
Done

**Comment by bergmark** [comment]
thanks!!

**Pull request merged**